src/StackExchange.Redis/Message.cs

```diff
@@ -60,8 +58,7 @@ internal abstract class Message : ICompletable
 
         private const CommandFlags AskingFlag = (CommandFlags)32,
                                    ScriptUnavailableFlag = (CommandFlags)256,
-                                   NeedsAsyncTimeoutCheckFlag = (CommandFlags)1024,
-                                   DemandSubscriptionConnection = (CommandFlags)2048;
+                                   DemandSubscriptionConnection = (CommandFlags)1024;
```

Comment:
Nit, I habitually avoid changing the numeric value of enums. Why take any risk of something thats hard to debug?

Reply:
Since these are all internal to-the-build it's more of a tidy, but I also have no strong feelings here - will migrate back!